CreateCommand.py

```diff
@@ -0,0 +1,18 @@
+import json
+from msilib import schema
+import os
+# provides functions for creating, removing a directory (folder), fetching its contents, changing and identifying the current directory
+from ICommand import ICommand
+from Keys import Keys
+class CreateCommand(ICommand):
+    def excute(schema):
+        with open('schema.json','r') as schema:
+            data = json.load(schema)
```

Comment:
Move this to constructor. you don't need to parse the shema in execute. execute should have the schema parsed which will be in constructor